presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,60 @@
+========================
+Release 328 (6 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated JOIN queries when the subquery on the
+  right produces no rows. (:issue:`1969`)
+* Fix incorrect handling of multi-byte characters when pattern is empty. (:issue:`2313`)
+* Fix failure when join criteria contains columns of different types. (:issue:`2320`)
+* Fix failure for complex outer join queries when dynamic filtering is enabled. (:issue:`2363`)
+* Improve support for correlated queries. (:issue:`1969`)
```

Comment:
can this be more specific or provide an example?

Reply:
It's hard to describe concisely. It's also one of those fixes that "unblocks" a bunch of other optimizations that allow for the decorrelation rules to work.